pbp.sh

```diff
@@ -1,27 +1,14 @@
-echo "$(tput setaf 5)Pulling and bundling consumer-layouts...$(tput sgr0)"
-cd consumer-layouts
-git pull
-bundle install
+#!/usr/bin/env bash
 
-echo "$(tput setaf 5)Pulling and bundling consumer-domain...$(tput sgr0)"
-cd ../consumer-domain
-git pull
-bundle install
+REPOS=(consumer-layouts consumer-domain consumer-session domain-values consumer-main)
```

Comment:
Yeah, this doesn't work... It only does the first one. And when I switch it to newlines and quotes, it has an issue with `cd` after the first one. I thought it was working, too, hence the merge. But I wasn't looking close enough.